app/src/main/java/com/phantom/onetapfacebookmodule/FacebookHook.java

```diff
@@ -38,16 +38,18 @@ protected void afterHookedMethod(XC_MethodHook.MethodHookParam hookParams) throw
                 try {
                     Uri uri = (Uri) hookParams.args[0];
                     ApplicationLogMaintainer.sendBroadcast(context, "Facebook Main URL : " + uri.toString());
-                    String stringUri = "";
```

Comment:
@Ashish-Bansal you're completely right, just thought about it in the toilet:) i'll fix it right away.